docs/tutorials/fundamentals/viewer.md

```diff
@@ -276,46 +251,30 @@ viewer.add_shapes()
 viewer.add_labels()
```

Comment:
This one actually raises an error:
```
TypeError                                 Traceback (most recent call last)
Cell In[16], line 6
      4 viewer.add_points()
      5 viewer.add_shapes()
----> 6 viewer.add_labels()

TypeError: add_labels() missing 1 required positional argument: 'data'
```
I'm not sure why at the moment, because the button does work, so this is a case where the GUI and code diverge for some reason—probably due to the fact that the size/dimenions of the labels would be undefined, while I guess the button makes an assumption.

Reply:
What line does this come from?

Reply:
I don't know how to correct the code or really even what the problem is.  I get the error on viewer.add_labels() also, as follows: 
TypeError: add_labels() missing 1 required positional argument: 'data'

Reply:
Yeah, I don't think there is any easy solution. 
This is a case where the code approach and GUI diverge.
The UI button is passing some sort of data to the `add_labels` method, while `viewer.add_labels()` obviously doesn't.
The other two layer types are fine without data though, so maybe this behavior is a bug?
Maybe @jni or @DragaDoncila have some ideas how to proceed?